# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ntop/no_etter_passive_os_fp.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur ntop-4.0.3-orig/Makefile.am ntop-4.0.3/Makefile.am
--- ntop-4.0.3-orig/Makefile.am	2010-12-19 22:09:55.360061877 +0100
+++ ntop-4.0.3/Makefile.am	2010-12-19 22:11:06.984061885 +0100
@@ -401,45 +401,7 @@
 
 # download the passive ethernet fingerprint database
 dnetter:
-	@echo ""
-	@echo ""
-	@echo "Preparing "
-	@echo ""
-
-	@if test -f $(ETTER_PASSIVE).old; then \
-	  echo "...Deleting prior file, $(ETTER_PASSIVE).old..."; \
-	  rm -rf $(ETTER_PASSIVE).old; \
-	fi;
-	@if test -f $(ETTER_PASSIVE); then \
-	  echo "...Moving current $(ETTER_PASSIVE) to $(ETTER_PASSIVE).old"; \
-	  mv $(ETTER_PASSIVE) $(ETTER_PASSIVE).old; \
-	fi;
-	@for file in $(ETTER_PASSIVE) $(ETTER_PASSIVE).gz \
-		$(ETTER_PASSIVE_FILE) $(ETTER_PASSIVE_FILE).gz; do \
-          if test -f $$file; then \
-	     echo "...Deleting prior file, $$file..."; \
-             rm -rf $$file; \
-          fi; \
-        done
-	@echo ""
-	@echo "...Downloading new file..."
-	@wget -O $(ETTER_PASSIVE_FILE) \
-		$(ETTER_PASSIVE_DOWNLOAD_FROM)/$(ETTER_PASSIVE_FILE)?$(ETTER_PASSIVE_DOWNLOAD_PARMS)
-	@echo ""
-	@echo "gziping downloaded file..."
-	@gzip $(ETTER_PASSIVE_FILE)
-	@echo ""
-	@echo ""
-	@if test -f $(ETTER_PASSIVE).old; then \
-	  echo -n "Old file lines were: "; \
-	  gunzip -c $(ETTER_PASSIVE).old | wc -l; \
-	fi;
-	@echo -n "New file lines are:  "
-	@gunzip -c $(ETTER_PASSIVE) | wc -l
-	@echo ""
-	@echo ""
-	@echo "New file is:"
-	@ls -l $(ETTER_PASSIVE)
+	@echo "download the passive ethernet fingerprint database manually"
 
 # ntop census
 census-fail:
